exampleCourse/questions/examplesPanels/question.html

```diff
@@ -0,0 +1,77 @@
+
+  <h5>Elements to control panel display</h5>
+
+  <p>There are four elements that can be used to control whether text is shown in each panel:</p>
+  <ul>
+      <li><b><tt>&lt;pl-question-panel&gt;</tt></b> only displays its contents in the question panel. This should be used for the question prompt.</li>
+      <li><b><tt>&lt;pl-submission-panel&gt;</tt></b> only displays its contents in the submission panel. This could be used to give specific feedback on a particular student submission.</li>
+      <li><b><tt>&lt;pl-answer-panel&gt;</tt></b> only displays its contents in the answer panel. This could be used to give information about the correct answer.</li>
+      <li><b><tt>&lt;pl-hide-in-panel&gt;</tt></b> has attributes that allow content to be hidden in one or more panels.</li>
+  </ul>
+
+  <p>In normal usage, we put the question prompt in a <tt>&lt;pl-question-panel&gt;</tt> element and all inputs requesting student input outside of this element.</p>
+
+  <h5>Example HTML</h5>
+
+  <p>As an example, this question contains the follow panel elements.  Click the "Save &amp; Grade" button to make the submission and answer panels appear.</p>
+<pre>
+&lt;pl-question-panel&gt;
+  &lt;p&gt;(A) This text is only shown in the question panel.
+&lt;/pl-question-panel&gt;
+
+&lt;pl-submission-panel&gt;
+  &lt;p&gt;(B) This text is only shown in the submission panel.
+&lt;/pl-submission-panel&gt;
+
+&lt;pl-answer-panel&gt;
+  &lt;p&gt;(C) This text is only shown in the answer panel.
+&lt;/pl-answer-panel&gt;
+
+&lt;pl-hide-in-panel question="True"&gt;
+  &lt;p&gt;(D) This text is hidden in the question panel, and shown in the others.
+&lt;/pl-hide-in-panel&gt;
+
+&lt;pl-hide-in-panel submission="True"&gt;
+  &lt;p&gt;(E) This text is hidden in the submission panel, and shown in the others.
+&lt;/pl-hide-in-panel&gt;
+
+&lt;pl-hide-in-panel answer="True"&gt;
+  &lt;p&gt;(F) This text is hidden in the answer panel, and shown in the others.
+&lt;/pl-hide-in-panel&gt;
+</pre>
+
+<h5>Output of above example</h5>
+</pl-question-panel>
+
+<pl-question-panel>
+  <p>(A) This text is only shown in the question panel.</p>
+</pl-question-panel>
+
+<pl-submission-panel>
+  <p>(B) This text is only shown in the submission panel.</p>
+</pl-submission-panel>
+
+<pl-answer-panel>
+  <p>(C) This text is only shown in the answer panel.</p>
+</pl-answer-panel>
+
+<pl-hide-in-panel question="True">
+  <p>(D) This text is hidden in the question panel, and shown in the others.</p>
+</pl-hide-in-panel>
+
+<pl-hide-in-panel submission="True">
+  <p>(E) This text is hidden in the submission panel, and shown in the others.</p>
+</pl-hide-in-panel>
+
+<pl-hide-in-panel answer="True">
+  <p>(F) This text is hidden in the answer panel, and shown in the others.</p>
+</pl-hide-in-panel>
```

Comment:
Maybe add an example with two states?  e.g.

`<pl-hide-in-panel question="True" answer = "True">`

and, then show it matches

`<pl-submission-panel>`?

Reply:
It’s cute that these match, but it doesn’t seem very practical? I wanted to focus on “here’s what to do to get the effect you want”, if that makes sense?